src/utilities/nanoid.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * A secure, URL-friendly, unique string ID generator function in TypeScript,
+ *
+ * This is used instead of the package "nanoid" as that package uses a module
+ * format that is not compatible with our build system.
```

Comment:
I can probably walk you through fixing this. If you look at the `transformIgnorePatterns` of the Jest config in CODAPv3 or CLUE package.json you'll see one place. But perhaps there is something beyond Jest that has to be changed.